TemplateApplication/TemplateApplicationDelegate.swift

```diff
@@ -77,7 +77,7 @@ class TemplateApplicationDelegate: SpeziAppDelegate {
         HealthKit {
             CollectSample(
                 HKQuantityType(.stepCount),
-                deliverySetting: .anchorQuery(.afterAuthorizationAndApplicationWillLaunch)
+                deliverySetting: .anchorQuery(.automatic)
```

Comment:
@NikolaiMadlener I just fixed this in my push to remove the deprecated API; not really worth a separate PR so I just pushed and added it here.

Reply:
thx!